jekyll/_cci2/env-vars-inject-with-api.adoc

```diff
@@ -4,7 +4,7 @@ version:
 - Server 3.x
 - Server 4.x
 ---
-= Inject environment variables with the CircleCI API
+= Inject Environment Variables with the CircleCI API
```

Comment:
I thought we weren't doing title case anymore since the blog/outer doesn't use it anymore either.